src/drivers/intel/cavs/ssp.c

```diff
@@ -712,6 +712,18 @@ static void ssp_stop(struct dai *dai, int direction)
 	spin_unlock(&dai->lock);
 }
 
+static void ssp_pause(struct dai *dai, int direction)
```

Comment:
Wont we need the same logic on the other SSP drivers since it's being called from dai.c ?

Reply:
Yes, I added changes to other platforms too.